pixeltable/exprs/row_builder.py

```diff
@@ -3,7 +3,7 @@
 import sys
 import time
 from dataclasses import dataclass
-from typing import Any, Dict, Iterable, List, Optional, Sequence, Set, Tuple
+from typing import Any, Iterable, Optional, Sequence
```

Comment:
Could you leave this untouched? I already made those changes in the media validation PR, and I'd like to avoid having to resolve all those merge conflicts.